doc/manpages/options.asciidoc

```diff
@@ -5,6 +5,37 @@ NAME
 ----
 options - a
 
+Description
+-----------
+
+Options are used for user configuration.
+You can set the builtin ones or declare your owns.
+
+The following commands are available to interact with them:
+
+----------------------------------------------
+declare-option [-hidden] <type> <name> [value]
+set-option [-add] <scope> <name> <value>
+update-option <scope> <name>
+unset-option <scope> <name>
+----------------------------------------------
+
+Refer to the 'scopes' documentation page for more information about
+scopes.
+
+To get the current value of an option:
```

Comment:
You mean having e.g. `|expansions|` in the document, and have the page be opened automatically when hitting `<ret>` (or clicking?) on it? It would be nice, also we could use a character other than `|`, if those sequences aren't interpreted by `a2x`.